data/collections/geoglam.json

```diff
@@ -0,0 +1,27 @@
+{
+  "id": "geoglam",
+  "type": "Collection",
+  "links": [],
```

Comment:
Should we add any links?

Reply:
Yes please could we add https://cropmonitor.org/